Updates guard documentation to latest SwiftLint rules (#9)
FundingCircle/swift-style-guide: README.md

guard let thingOne = thingOne else { return }
```

> * **3.11.8** Always left an empty line after every single guard

Reviewer: ```suggestion
* **3.11.8** There should always be an empty line after every guard.
```

suggestion to make tone a bit more consistent with the rest of the doc.

Thanks for doing this!

Author: make sense, done 👍
